notebooks/data-sources/TestGrid/testgrid_metadata_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
      "name": "stdout",
```

Comment:
<p>The outcome of this notebook should be a recommendation for what values are worth additional attention, possible analysis and that we might want to collected regularly, vs which seem to be noise or unused place holders in the testgrid schema. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='169'/>

Reply:
Yeah, we could probably add a "summary of findings" or "conclusion" section elaborating on the above.